docs/opentracing/opentracing.md

```diff
@@ -0,0 +1,36 @@
+Các hệ thống truy vết hiện hành (Zipkin,Dapper, HTrace, X-Trace….) được sinh ra nhằm giải quyết những vấn đề trên bằng cách thêm công cụ truy vết vào tầng ứng dụng bằng việc sử dụng những API không tương thích. Các nhà phát triển sẽ không dễ dàng khi phải tìm một công cụ truy vết nào để sử dụng cho một hệ thống sử dụng nhiều ngôn ngữ lập tình của họ, tuy nhiên những APIs của công truy vết tầng ứng dụng này cũng có những ý nghĩa và công dụng nhất định của nó.
+OpenTracing: bằng cách cung cấp các APIs thông nhất, nhanh gọn cho hầu hết các nền tảng phổ biến. OpenTracing giúp các nhà phát triển thêm (hoặc chuyển đổi) truy vết một cách dễ dàng.
+
+## 2.OpenTracing API
+OpenTracing định nghĩa một API thông qua đó thiết bị ứng dụng có thể đăng nhập  dữ liệu vào pluggable tracer. Nhìn chung thì OpenTracing không đảm bảo về cách dữ liệu sẽ được xử lý bởi một vết cơ bản. Vì vậy, vấn đề đặt ra là loại dữ liệu nào nên được cũng nên được cung cấp cho các API để đảm bảo tốt nhất khả năng tương thích giữa các tracer.
+
+## 3. Những project đang tương thích
+-	Go [(opentracing-go)]( https://github.com/opentracing/opentracing-go)
+-	Python [(opentracing-python)]( https://github.com/opentracing/opentracing-python)
+-	Javascript [(opentracing-javascript)]( https://github.com/opentracing/opentracing-javascript)
+-	Java [(opentracing-java)]( https://github.com/opentracing/opentracing-java)
+-	C# [(opentracing-csharp)](https://github.com/opentracing/opentracing-csharp)
+-	Objective-C [(opentracing-objc)]( https://github.com/opentracing/opentracing-objc)
+-	C++ [(opentracing-cpp)](https://github.com/opentracing/opentracing-cpp)
+-	Ruby [(opentracing-ruby)]( https://github.com/opentracing/opentracing-ruby)
+-	PHP (sắp tới)
+## 4. Các client được hỗ trợ cùng với các ngôn ngữ.
+* Zipkin: hỗ trợ nhiều ngôn ngữ như [Brave(Zipkin Java)]( https://github.com/openzipkin/brave-opentracing) đến [Zipkin-go-opentracing]( https://github.com/openzipkin/zipkin-go-opentracing).
+* Jaeger: là một hệ thống truy tìm phân tán của Uber, được xây dựng từ OpenTracing từ khi bắt đầu, Jaeger bao gồm các client một số ngôn ngữ như [Java]( https://github.com/uber/jaeger-client-java), [Go]( https://github.com/uber/jaeger-client-go), [Python]( https://github.com/uber/jaeger-client-python), [Node.js]( https://github.com/uber/jaeger-client-node).
+* Appdash: là một hệ thống do tìm phân phối dựa trên Golang-based, ban đầu đưuọc phát triển và có nguồn mở từ sourcegraph. Opentracing sử dụng Appdash như một phụ trợ.
+* LightStep: tương thích với Opentracing dành cho Go, Python, Javascript, Objective-C, Java,PHP, Ruby và C++.
+* Hawkular:  hỗ trợ client [Java](https://github.com/hawkular/hawkular-client-java), [Go](https://github.com/hawkular/hawkular-client-go), [Ruby](https://github.com/hawkular/hawkular-client-ruby), [Python](https://github.com/hawkular/hawkular-client-python).
+* Sky-walking: hỗ trợ client Java.
+## 5. Data model của OpenTracing
```

Comment:
![The opentracing software architecture](https://cloud.githubusercontent.com/assets/25746756/25061926/4baa3b7e-21eb-11e7-8032-f0be30dd5d2c.png)